themes/hugo-theme-learn/layouts/_default/sitemap.xml

```diff
@@ -0,0 +1,32 @@
+{{ printf "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"yes\" ?>" | safeHTML }}
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml">
+{{- $products := .Site.Data.products -}}
+{{ range .Data.Pages }}
+{{- $isStableVersion := true -}}
```

Comment:
Why don't we generate the sitemap for _all_ versions?

Reply:
@xrstf this was decision of Sebastian.